akka-docs/src/main/paradox/general/actor-systems.md

```diff
@@ -75,7 +75,7 @@ a distributed application.
     requests) in an event-driven manner. Actors should not block (i.e. passively
     wait while occupying a Thread) on some external entity—which might be a
     lock, a network socket, etc.—unless it is unavoidable; in the latter case
-    see below.
```

Comment:
This "below" was moved to the dispatchers.md on version 2.5. Now it has a proper link pointing to this information